mapflow/mapflow.py

```diff
@@ -1503,26 +1503,25 @@ def post_processing_callback(self, _: QNetworkReply, processing_name: str) -> No
         )
 
     def set_processing_limit(
-        self,
-        response: Optional[QNetworkReply] = None,
-        response_data: Optional[dict] = None) -> None:
+            self,
+            response: Optional[QNetworkReply] = None,
```

Comment:
Why do we need these parameters? I think we should get rid of them and change all the call signatures.